.travis.yml

```diff
@@ -4,7 +4,8 @@ python:
 
 install:
   - pip install -r requirements.txt
-
+before_script:
+  - "flake8"
```

Comment:
Does this need to be in quotes

Reply:
I was following the flake8 documentation but let me verify